Code/Tools/AssetProcessor/native/AssetManager/AssetRequestHandler.cpp

```diff
@@ -104,29 +104,29 @@ namespace
             {
                 case AssetChangeReportRequest::ChangeType::CheckMove:
                 {
-                    auto resultCheck = relocationInterface->Move(messageData.m_message->m_fromPath, messageData.m_message->m_toPath);
+                    auto resultCheck = relocationInterface->Move(messageData.m_message->m_fromPath, messageData.m_message->m_toPath, true, true, false, true);
 
                     BuildReport(relocationInterface, resultCheck, lines);
                     break;
                 }
                 case AssetChangeReportRequest::ChangeType::Move:
                 {
                     auto resultMove = relocationInterface->Move(
-                        messageData.m_message->m_fromPath, messageData.m_message->m_toPath, false, true, true, true);
+                        messageData.m_message->m_fromPath, messageData.m_message->m_toPath, false, true, false, true);
 
                     BuildReport(relocationInterface, resultMove, lines);
                     break;
                 }
                 case AssetChangeReportRequest::ChangeType::CheckDelete:
                 {
-                    auto resultCheck = relocationInterface->Delete(messageData.m_message->m_fromPath);
+                    auto resultCheck = relocationInterface->Delete(messageData.m_message->m_fromPath, true, true, false);
 
                     BuildReport(relocationInterface, resultCheck, lines);
                     break;
                 }
                 case AssetChangeReportRequest::ChangeType::Delete:
                 {
-                    auto resultDelete = relocationInterface->Delete(messageData.m_message->m_fromPath, false, true, true);
+                    auto resultDelete = relocationInterface->Delete(messageData.m_message->m_fromPath, false, true, false);
 
                     BuildReport(relocationInterface, resultDelete, lines);
```

Comment:
Oh my eyes 😖 Those `bool` parameters... it's impossible to see what's going on here at a glance... is there any chance we could improve this while we're here and ideally make an issue to follow-up to make this interface clearer by moving to an enum or splitting up the functionality in that call?

(I know it was like this before your change but while we're here it seems like a disservice to not try and make thing a bit better for the next person who comes along 😅)